src/compas_fab/robots/robot.py

```diff
@@ -557,7 +638,23 @@ def get_position_by_joint_name(self, configuration, joint_name, group=None):
         return configuration.values[names.index(joint_name)]
 
     def _scale_joint_values(self, values, names, scale_factor, group=None):
-        """Scales the scaleable joint values with scale_factor.
+        """Scales the scaleable joint values with specified scale factor.
```

Comment:
A bit redundantly scalable, right? ;)

```suggestion
        """Scales joint values to the specified factor. Only certain joint types are scalable.
```